submissions/jfrank/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "university",
+  "version": "0.0.1",
+  "description": "Viget / hapijs University",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "engines": {
+    "node": "4.0.0"
+  },
+  "scripts": {
+    "start": "node lib/index",
+    "test": "lab"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/jeremyfrank/university.git"
+  },
+  "author": "Jeremy Frank",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jeremyfrank/university/issues"
+  },
+  "homepage": "https://github.com/jeremyfrank/university#readme",
+  "dependencies": {
+    "code": "2.1.0",
+    "lab": "8.0.1"
+  }
```

Comment:
I think `npm install x --save` is just directing it to update your package.json w/ an item for that. What Nate's config setting does is make it always save the exact version, not the `^` type version. 

@nhunzaker where do you edit your npm config?